client/src/pages/ResourcePage/ResourcePage.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../theme";
+
+.resources.page {
+    .static-page-body {
+        width: 935px;
+        max-width: 935px;
+
+        .row:last-child {
+            border: none;
+            margin: 0;
+        }
+    }
+
+    .field-name {
+        display: flex;
+        align-items: center;
+        i {
+            transform: translateY(-2px);
+            cursor: pointer;
+            color: $secondary;
+            user-select: none;
+            &:hover {
+                color: darken($secondary, 15%);
+                transform: scale(1.1) translateY(-2px);
+            }
+            &:active {
+                color: darken($secondary, 30%);
+            }
+        }
+    }
+
+    .field-content {
+        padding-top: 0.75rem;
+
+        p {
+            margin-bottom: 0.5rem;
+        }
+
+        .important {
+            text-decoration: none !important;
+            font-weight: 500;
+            color: $secondary;
```

Comment:
I think she wanted these italicized too, not 100% on that

Reply:
I'll run it by her today

Reply:
This is all set, no need to change the styling.